examples/b2c/checkout/integrations/classes/B2CCheckInventorySample.cls

```diff
@@ -1,6 +1,10 @@
 // This must implement the sfdc_checkout.CartInventoryValidation interface
 // in order to be processed by the checkout flow and used for your Check Inventory integration.
 global class B2CCheckInventorySample implements sfdc_checkout.CartInventoryValidation {
+    // To access the service below, you may need to add https://example.com in Setup | Security | Remote site settings.
```

Comment:
Update this comment in every class to say that you MUST change this to be your service or you must launch your own Heroku service.  I'd suggest we make the original heroku app being available.

Reply:
Then add a separate comment to remind people to provide their own app url in remote site settings.